src/wasm-interpreter.h

```diff
@@ -4103,7 +4103,7 @@ class ModuleRunnerBase : public ExpressionRunner<SubType> {
   // can use it (refactor?)
   Literals callFunctionInternal(Name name, Literals arguments) {
     if (callDepth > maxDepth) {
-      externalInterface->trap("stack limit");
+      hostLimit("stack limit");
```

Comment:
Good catch...

Reply:
Now that we support `assert_exhaustion`, we actually have test coverage for this!